src/html/index.html

```diff
@@ -187,11 +187,22 @@ <h2>PWM Output</h2>
 							<li><b>Input:</b> Input channel from the handset</li>
 							<li><b>Invert:</b> Invert input channel position</li>
 							<li><b>750us:</b> Use half pulse width (494-1006us) with center 750us instead of 988-2012us</li>
-							<li><b>Failsafe:</b> Absolute position to set the servo on failsafe
+							<li><b>Failsafe</b>
 								<ul>
-									<li>Does not use "Invert" flag</li>
-									<li>Value will be halved if "750us" flag is set</li>
-									<li>Will be converted to binary for "On/Off" mode (>1500us = HIGH)</li>
+									<li>Set Position - Absolute position to set the servo on failsafe
+										<ul>
+											<li>Does not use "Invert" flag</li>
+											<li>Value will be halved if "750us" flag is set</li>
+											<li>Will be converted to binary for "On/Off" mode (>1500us = HIGH)</li>
+										</ul>
+									</li>
+									<li>No Pulse - Stop pulsing
+										<ul>
+											<li>Unpowers servos</li>
+											<li>May disarm ESCs</li>
+										</ul>
+									</li>
+									<li>Hold Position - Uses last received channel position</li>
```

Comment:
"Set Position" and "Hold Position" should use the text from the combobox, as this is text is to document what the settings are and therefore should match. I think the descriptions could be actions instead of mixing actions and subjects.
* Failsafe Behavior (<-- add "behavior")
  * Set Pos - Set servo to absolute "Failsafe Pos"
  * No Pulses - Stop sending pulses
  * Hold Pos - Continue sending last received channel position

Reply:
If you want consistency, there's also the section of settings meant for SBUS, that uses the words "No Pulses" and "Last Position", and the format is different from mine, it reads like

```
"Last Position" continues to send the last received channel data along with the FAILSAFE bit set
```

I can copy that continuous-action-sentence if you'd like (without the hyphen), change the word "Hold" to "Last", and also use the full word "Position". I used the shortened "Pos" because the web-ui is already running out of space on a smartphone lol that was a desperate attempt at trying to make the table narrower.

I'll screw around with the CSS later to see if I can make the table behave at least a bit acceptable on my own phone screen, right now I can't actually see what option I have selected in any of the comboboxes. The solution is likely just imposing minimum widths on the elements, and then letting it overflow beyond screen border, and then rely on the browser's horizontal scrolling.

Reply:
I hear ya on the phone UI not fitting. It is kinda a problem. It's ok if you want to use Pos / Position, but they just need to match between the dropdown / documentation so it is consistent.

I had forgotten about the SBUS block because I had CRSF set when I tested, but you're right it should be close to the same. Maybe "Hold pos - Continue to send the last received channel data" (note the tense is changed from Continues to Continue to match the tense of the other phrases, but I am ok with whatever you like so long as all 3 have the same tense)

Reply:
I made changes, it aligns the wording with the wording in the dropdown, and uses the same style as the sbus section

I also made CSS changes to give the table elements a minimum width

I tried using min-width:fit-content but that seem to work only on desktop, on mobile it seems like the fit-content doesn't work for the select element
![Screenshot_20240202-205057_Chrome](https://github.com/ExpressLRS/ExpressLRS/assets/1427911/6f742ca7-17f7-4b41-92e3-b5a4e2558d9e)
![Screenshot_20240202-205612_Chrome](https://github.com/ExpressLRS/ExpressLRS/assets/1427911/ced65e17-1b8c-4cad-9365-cb0e2c4941f0)

I think it's much more useable than before